tests/providers/google/cloud/operators/test_automl.py

```diff
@@ -324,41 +350,38 @@ def test_execute(self, mock_hook):
         dataset = copy.deepcopy(DATASET)
         dataset["name"] = DATASET_ID
 
-        op = AutoMLTablesUpdateDatasetOperator(
-            dataset=dataset,
-            update_mask=MASK,
-            location=GCP_LOCATION,
-            task_id=TASK_ID,
-        )
-        op.execute(context=mock.MagicMock())
-        mock_hook.return_value.update_dataset.assert_called_once_with(
-            dataset=dataset,
-            metadata=(),
-            retry=DEFAULT,
-            timeout=None,
-            update_mask=MASK,
+        with pytest.raises(AirflowProviderDeprecationWarning) as err:
+            AutoMLTablesUpdateDatasetOperator(
+                dataset=dataset,
+                update_mask=MASK,
+                location=GCP_LOCATION,
+                task_id=TASK_ID,
+            )
+        assert str(err.value).startswith(
+            "Call to deprecated class AutoMLTablesUpdateDatasetOperator. "
+            "(Class `AutoMLTablesUpdateDatasetOperator` is deprecated and will be removed after"
```

Comment:
It would be nicer to use the `match` keyword instead:
https://docs.pytest.org/en/7.1.x/how-to/assert.html

You are welcome to apply it to similar assertions :)